metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -209,136 +196,6 @@ public void testUnregisterBroker() throws Throwable {
         }
     }
 
-    static class MockSnapshotWriterBuilder implements Function<Long, SnapshotWriter> {
```

Comment:
I don't want to remove the snapshot handling code or tests that we have for the controller. We should find a way to keep this test working before we merge this PR.